config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/mechanic/:id", to: "mechanic#show"
+  get "/mechanic/:id/new", to: "mechanic#new"
```

Comment:
This `"new"` route is incorrect and is not RESTful. This should be `get "/mechanic/:id/rides/new", to: "mechanic_rides#new"` because we creating a new mechanic ride, not a new mechanic. If we were creating a new mechanic, the route would be `get "/mechanic/new", to: "mechanic#new"`, without the `:id` piece.

I would suggest reviewing your notes on REST.